### PR TITLE
Updates example for Set-MgBetaApplicationConnectorGroupByRef

### PR DESCRIPTION
**Pull request by timayabi2020**
Updates exampleto match the powershell snippet in the API reference https://learn.microsoft.com/graph/api/connectorgroup-post-applications?view=graph-rest-1.0

### CODE REVIEW

File: src/Applications/beta/examples/Set-MgBetaApplicationConnectorGroupByRef.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-	"@odata.id" = "https://graph.microsoft.com/onPremisesPublishingProfiles/applicationproxy/connectorGroups/{id}"
+	"@odata.id" = "https://graph.microsoft.com/beta/onPremisesPublishingProfiles/applicationproxy/connectorGroups/{id}"
 }
+
 Set-MgBetaApplicationConnectorGroupByRef -ApplicationId $applicationId -BodyParameter $params
+
 ```
 This example shows how to use the Set-MgBetaApplicationConnectorGroupByRef Cmdlet.
-To learn about permissions for this resource, see the [permissions reference](/graph/permissions-reference).
+
```
Comment (calebkiage) [inline src/Applications/beta/examples/Set-MgBetaApplicationConnectorGroupByRef.md:15]: Why did you remove the notice on permissions?

Comment (timayabi2020) [reply src/Applications/beta/examples/Set-MgBetaApplicationConnectorGroupByRef.md:15]: That bit is general and not specific to the path related to this command. We handled that issue by including the exact permissions table during weekly reference docs refresh. Have a look at this https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.beta.applications/set-mgbetaapplicationconnectorgroupbyref?view=graph-powershell-beta

image

**Review by calebkiage — APPROVED** [review]
